### ISSUE TITLE
resources

### ISSUE THREAD

**Issue by MaryBN** [MaryBN/mary#1]
<!DOCTYPE html>
<html lang="en">

<head>
	<title>resources</title>
	
	<meta charset="utf-8">
	<meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
  <style>
    /* Set black background color, white text and some padding */
    footer {
      background-color: #555;
      color: white;
      padding: 15px;
    }
  </style>
</head>
<body>

<nav class="navbar navbar-inverse">
  <div class="container-fluid">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#myNavbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="#">Logo</a>
    </div>
    <div class="collapse navbar-collapse" id="myNavbar">
      <ul class="nav navbar-nav">
        <li><a href="#">Home</a></li>
        <li class="active"><a href="#">Resources</a></li>
		<li><a href="donations.html">Donation</a></li>
		<li><a href="donations.html">Getting Involved</a></li>
      </ul>
      <form class="navbar-form navbar-right" role="search">
        <div class="form-group input-group">
          <input type="text" class="form-control" placeholder="Search..">
          <span class="input-group-btn">
            <button class="btn btn-default" type="button">
              <span class="glyphicon glyphicon-search"></span>
            </button>
          </span>
        </div>
      </form>
      <!ul class="nav navbar-nav navbar-right"> 															 <!My account log in thingy>
        <!li><!a href="#"><!span class="glyphicon glyphicon-user"><!/span> My Account<!/a><!/li>
      <!/ul>
    </div>
  </div>
</nav>
  
<div class="container text-center">
  <div class="row">
    <div class="col-sm-3 well">
      <div class="well">
        <p><a href="women.html">WOMEN</a></p>
        <!img src="bird.jpg" class="img-circle" height="65" width="65" alt="Avatar">
      </div>
      <div class="well">
        <p><a href="#">SHELTER</a></p>
		</div>
	  <div class="well">
		<p><a href="jobs.html">JOBS</a></p>
        </div>
      <div class="well">
		<p><a href="counseling.html">COUNSELING</a></p>
        </div>
    </div>
    <div class="col-sm-7">
    
      <div class="row">
        <div class="col-sm-12">
          <div class="panel panel-default text-left">
            <div class="panel-body">
              <p style="font-size:200%;" contenteditable="true">Upcoming Events</p>
            </div>
          </div>
        </div>
      </div>
      
      <div class="row">
        <div class="col-sm-3">
          <div class="well">
           <p><a href="#">User X</p>
		   <img src="#" class="img-circle" height="55" width="55" alt="pic">
          </div>
        </div>
        <div class="col-sm-9">
          <div class="well">
            <p>Blablabla!</p>
          </div>
        </div>
      </div>
      <div class="row">
        <div class="col-sm-3">
          <div class="well">
           <p><a href="#">User Y</p>
		   <img src="#" class="img-circle" height="55" width="55" alt="Avatar">
           </div>
        </div>
        <div class="col-sm-9">
          <div class="well">
            <p>HEY SOLOME.</p>
          </div>
        </div>
      </div>
      <div class="row">
        <div class="col-sm-3">
          <div class="well">
           <p><a href="#">Event X</p>
           <img src="#" class="img-circle" height="55" width="55" alt="Avatar">
          </div>
        </div>
        <div class="col-sm-9">
          <div class="well">
            <p></p>
          </div>
        </div>
      </div>
      <div class="row">
        <div class="col-sm-3">
          <div class="well">
           <p><a href="#">User P</p>
		   <img src="#" class="img-circle" height="55" width="55" alt="Avatar">
          </div>
        </div>
        <div class="col-sm-9">
          <div class="well">
            <p></p>
          </div>
        </div>
      </div>
    </div>
    <div class="col-sm-2 well">
      <p>Our organization has help for your specif need.</p>
      </div>
    </div>
  </div>
</div>

<footer class="container-fluid text-center">
  <p>For more up coming events go to get involved's web page!</p>
</footer>

</body>
</html>
